.github/CODEOWNERS

```diff
@@ -43,3 +49,6 @@ Dockerfile.ci @potiuk @ashb
 /dev/ @potiuk @ashb @kaxil
 /provider_packages/ @potiuk @ashb
 /scripts/ @potiuk @ashb
+
+# Helm Chart
+/chart/ @kaxil
```

Comment:
/chart/ is already set in the kubernetes section and should be merged together.

Reply:
Oh right -- good catch